src/com/facebook/buck/swift/SwiftBuckConfig.java

```diff
@@ -36,6 +37,18 @@ public SwiftBuckConfig(BuckConfig delegate) {
     return value.map(input -> Splitter.on(" ").split(input.trim()));
   }
 
+  public Optional<ImmutableList<String>> getFlags(String field) {
```

Comment:
The method above (`getFlags`) should just be implemented by calling `getFlags(COMPILER_FLAGS_NAME)`.